src/main/java/net/dv8tion/jda/api/events/entitlement/EntitlementUpdateEvent.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.api.events.entitlement;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Entitlement;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Indicates a user's subscription has renewed for the next billing period.
+ * The {@link Entitlement#getEndsAt() endsAt} field will have an updated value with the new expiration date.
+ *
+ * <p><b>Notice</b><br>
+ * If a user's subscription is cancelled, you will not receive an {@link EntitlementDeleteEvent EntitlementDeleteEvent}.
+ * <br>Instead, you will simply not receive an {@link EntitlementUpdateEvent EntitlementUpdateEvent} with a new {@link Entitlement#getEndsAt() endsAt} date at the end of the billing period.
```

Comment:
This is a very strange description, and it is difficult for me to understand what it tries to tell me.

Reply:
I've asked around and it seems that this sentence means the cancellation disables the renew event. I think we can land on a better wording than what the official docs use. Something like:

> The ending date is updated for active subscriptions at the end of a billing period, to indicate a renewal. When a subscription is cancelled, the ending date will not be updated again as it is not renewed for the next billing period. Cancellation does not cause an immediate event and cancelled subscriptions do not delete an existing entitlement, since it is still active for the current billing period.

Reply:
Alright 👍🏼